src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -23,7 +23,9 @@
  */
 class Builder
 {
-    use BuildsQueries, Concerns\QueriesRelationships, ForwardsCalls;
+    use BuildsQueries,
+        ForwardsCalls,
+        Concerns\QueriesRelationships;
```

Comment:
Why one trait per line here? All other changes in this PR have all traits in a single line.

Besides, as of now Laravel is still following outdated PSR-2 code standard and it doesn't mention any rules towards trait use statements. 

And in PSR-12 each trait use statement must be on separate line with it's own `use` operator, so if we plan to move to PSR-12 in the future all traits use cases in entire code base will have to be changed anyway.

Reply:
We don't follow PSR-12 at the moment.

Reply:
> Why one trait per line here? All other changes in this PR have all traits in a single line.
Due to my observation throughout the codebase, I noticed two patterns that we're following when using traits. One is when using Concerns\* traits and the other is when not using Concerns\* traits

When using Concerns\* traits, we put one trait per line
For instance: 
+ src\Illuminate\Console\Command.php
+ src\Illuminate\Database\Connection.php
+ src\Illuminate\Database\Eloquent\Model.php
+ src\Illuminate\Foundation\Testing\TestCase.php
+ ...
So what I try to do is maintaining those two conventions, to improve consistency.

> And in PSR-12 each trait use statement must be on separate line with it's own use operator, so if we plan to move to PSR-12 in the future all traits use cases in entire code base will have to be changed anyway.
I'm so happy that you pointed it out. I did start this PR by re-formatting the entire code base following the PSR-12. However, this is my first time contribute to an open-source project, I think it would be better to start small, so I decided just to fix some minor inconsistencies first. If thing go well, I would suggest to do the work that you mentioned.